crates/pkgcraft-tools/src/subcmds/pkg/source.rs

```diff
@@ -463,9 +470,11 @@ fn source(targets: PkgTargets, cmd: &Command) -> anyhow::Result<ExitCode> {
     let stdout = io::stdout().lock();
     let mut out = cmd.format.make_output(stdout);
 
-    for result in pkgs.par_map(func).jobs(cmd.jobs) {
-        match result {
-            Ok(value) => {
+    let result: pkgcraft::Result<Vec<_>> = pkgs.par_bridge().map(func).collect();
```

Comment:
As this can technically be run across the entire repo, I'd rather not collect results into a vector.